src/daft-plan/src/physical_plan.rs

```diff
@@ -32,6 +34,7 @@ pub enum PhysicalPlan {
     TabularScanParquet(TabularScanParquet),
     TabularScanCsv(TabularScanCsv),
```

Comment:
So the other `TabularScan*` will no longer be used after introducing `TabularScan` ?

Reply:
I believe so, might need @clarkzinzow to confirm here

Reply:
Yep, once we remove the non-scan operator + micropartition path, these physical ops and their translation will be deleted!